cmd/shortener/main.go

```diff
@@ -1,37 +1,70 @@
+// Главный модуль main для запуска HTTP сервиса.
 package main
 
 import (
 	"fmt"
 	"log"
 	"net/http"
+	_ "net/http/pprof"
 	"os"
 	"os/signal"
+	"runtime"
 	"syscall"
 
-	"github.com/PerfectStepCoder/shorturl/cmd/shortener/config"
 	hdl "github.com/PerfectStepCoder/shorturl/internal/handlers"
 	"github.com/PerfectStepCoder/shorturl/internal/storage"
 	"github.com/go-chi/chi/v5"
+
+	"github.com/PerfectStepCoder/shorturl/cmd/shortener/config"
 )
 
+// Глобальные переменные сборки
+var buildVersion = "N/A" // версия продукта
+var buildDate = "N/A"    // дата сборки
+var buildCommit = "N/A"  // коммит сборки
+
+// mainStorage - хранилище для записи и чтения обработанных ссылок.
 var mainStorage storage.PersistanceStorage
 
+// lengthShortURL — константа длина генерируемых коротких ссылок.
 const lengthShortURL = 10
 
 func main() {
+
+	fmt.Printf("Build version: %s\n", buildVersion)
+	fmt.Printf("Build date: %s\n", buildDate)
+	fmt.Printf("Build commit: %s\n", buildCommit)
+
 	// Канал для получения сигналов
 	sigs := make(chan os.Signal, 1)
 	// Уведомлять о сигнале interrupt (Ctrl+C) и сигнале завершения
 	signal.Notify(sigs, syscall.SIGINT, syscall.SIGTERM)
 	// Создаем канал для уведомления о завершении работы
 	done := make(chan bool, 1)
 
+	// Для обработки удаления urls
+	inputCh := make(chan []string, 10000) // TODO вынести 10000 в переменные окружения
```

Comment:
Все правильно, не стоит оставлять "магические числа" в коде, стоит реализовать его как именованую константу.
```suggestion
	inputCh := make(chan []string, 10000) // TODO вынести 10000 в переменные окружения
```

Reply:
Готово, поправил

Reply:
По-моему еще одно место осталось.